### DISCUSSION TITLE
Issue with Livewire Dev Tools Enabled Despite Debug Mode Set to False

### DISCUSSION THREAD

**Discussion by Bagmaz** [livewire/livewire#8074] [1. Help]
Hello Livewire Team,

I hope this message finds you well. I am currently facing an issue with Livewire on my Laravel 10 project, where the Livewire Dev Tools seem to be enabled in the browser despite having the APP_DEBUG variable set to false in my .env file.

Here are the details of my setup:

Laravel Version: 10
Livewire Version: 3
Deployment: Laravel Forge
I have ensured that the APP_DEBUG variable in my .env file is set to false, but when I check the browser, the Livewire Dev Tools are still enabled. This is unexpected behavior, and I'm not sure why it's happening.

Could you please guide me on how to disable the Livewire Dev Tools when the APP_DEBUG is set to false?

Thank you for your assistance.

Best regards,
Hamid

**Comment by joshhanley** [comment]
@Bagmaz we don't maintain the Livewire dev tools, so you're best to raise an issue on the repo of whoever maintains it 🙂
